### DISCUSSION TITLE
Deployment of Hydrogen app failing

### DISCUSSION THREAD

**Discussion by karlssonemma** [vercel/community#847] [Help] [answered]
Any ideas to what I can do to try to fix this? Runs well locally.

<img width="1252" alt="Screenshot 2022-09-13 at 11 32 36" src="https://user-images.githubusercontent.com/65256196/189868633-aefa3b2b-577f-4839-8bda-6952b31afd1f.png">

**Comment by swarnava** [comment, marked as answer]
Hey, could you share deployment ID/link? was it working before?

**Reply by karlssonemma** [reply]
Thank you, but I just managed to finally get it working 🥳

**Reply by swarnava** [reply]
Amazing! Have a great rest of your day! :)
